tests/ballerina-compiler-api-test/src/test/resources/test-src/symbols/symbol-package-id/main.bal

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Simple types
+type UserNil null;
```

Comment:
```suggestion
type UserNil ();
```

Reply:
Fixed in f20d0cbd1e0c6cd4d6b5714b31f5d81e88bf4fde.